test/commands/scanner/e2e.cpd.test.ts

```diff
@@ -1,5 +1,6 @@
 import { expect } from "@salesforce/command/lib/test";
-import { setupCommandTest } from "../../TestUtils";
+// @ts-ignore
```

Comment:
The IDE complains when you have imports from files that aren't part of the tsconfig's `include` property. Adding this directive here and elsewhere to make that annoying complaint go away.

Reply:
I wonder if adding the test directory to the tsconfig's include would have negative side effects. I don't know typescript enough but I would have assumed it would be included along with the src directory. If it makes to do so, i'd prefer that over having all these ts-ignore comments.

Reply:
I will look into this and get back to you.